lib/stat_tracker.rb

```diff
@@ -7,6 +7,7 @@
 require './lib/season_stats'
 
 class StatTracker
+    attr_reader :games, :teams, :game_teams
```

Comment:
not sure if we need these attr_readers: unless you somehow are incorporating them in testing

Reply:
I needed them for writing tests and some of the methods but didn't use the :game_teams attr_reader.

Reply:
Smart! That was a clever way to address the issue.